setup.py

```diff
@@ -157,6 +173,18 @@ def get_numpy_status():
     return numpy_status
 
 
+def generate_cython():
+    cwd = os.path.abspath(os.path.dirname(__file__))
+    print("Cythonizing sources")
+    p = subprocess.call([sys.executable,
```

Comment:
stupid question: is there a reason to do subprocess instead of importing and calling ``main``?